Dockerfile-infinityhub-hpc

```diff
@@ -0,0 +1,88 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# MAY NOT BE IMPORTANT ANYMORE
```

Comment:
is it important?

Reply:
I guess we can remove it and if something breaks then we add it back in.